src/test/java/com/hedvig/notificationService/customerio/GetMemberPickedLocale.kt

```diff
@@ -0,0 +1,39 @@
+package com.hedvig.notificationService.customerio
+
+import com.hedvig.notificationService.serviceIntegration.memberService.MemberServiceClient
+import com.hedvig.notificationService.serviceIntegration.memberService.PickedLocale
+import io.mockk.every
+import io.mockk.mockk
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+import org.springframework.http.ResponseEntity
+import java.io.Serializable
+import java.util.Locale
+
+@RunWith(Parameterized::class)
+class GetMemberPickedLocale(val pickedLocale: String, val parsedLocale: Locale) {
```

Comment:
Maybe I'm old fashioned but even though I like the naming of what the test actually does, I would prefer if it ends with the word "Test" so you know it is a test when searching. 
```suggestion
class GetMemberPickedLocale(val pickedLocale: String, val parsedLocale: Locale) {
```
```suggestion
class GetMemberPickedLocaleTest(val pickedLocale: String, val parsedLocale: Locale) {
```

Reply:
Yes I like it better to.